frontend/components/site/siteBuilds.jsx

```diff
@@ -89,41 +92,56 @@ function branchLink(build) {
 
 function summarizeTaskResults(build) {
```

Comment:
This could be its own child component too.

Reply:
does thumbs up here mean do it now?

Reply:
Just general agreement, no need to add now